build.csx

```diff
@@ -0,0 +1,217 @@
+#load "packages/simple-targets-csx.5.2.0/simple-targets.csx"
+
+#r "System.Xml.Linq"
+
+//using System.Runtime.Serialization.Json;
+//using System.Xml;
+//using System.Xml.Linq;
+using static SimpleTargets;
+
+static readonly var  solutionName = "FakeItEasy";
+
+static readonly var solution = "./" + solutionName + ".sln";
+static readonly var packagesDir = Path.GetFullPath("packages");
+static readonly var repoUrl = "https://github.com/FakeItEasy/FakeItEasy";
+static readonly var versionAssembly = "./src/FakeItEasy/bin/Release/FakeItEasy.dll";
+static readonly IEnumerable<string> nuspecs = Directory.GetFiles("./src", "*.nuspec");

```

Comment:
None of these variables need to be declared with the explicit type `IEnumerable<string>`. They can all be `var`.